### PR TITLE
addon-options: Make shortcuts in storybook optional

### PR DESCRIPTION
**Pull request by PunitGr**
Issue: #3196 

## What I did
I've added an option for `enabling/disabling` keyboard shortcuts.

## How to test
One can try this feature by updating the storybook `config.js` as:
```javascript
setOptions({
    ...
    enableShortcuts: false, // true by default
})

This will disable the shortcuts.
```

### CODE REVIEW

**Comment by Hypnosphi** [comment]
Should we remove shortcuts help button and popup when shortcuts are disabled?

**Comment by PunitGr** [comment]
Yeah, that makes sense to disable it. Or we can add a text saying that `shortcuts` have been disabled.

**Comment by PunitGr** [comment]
Hey @Hypnosphi, Removed the shortcut help button when `enableShortcuts: false`.
screen shot 2018-03-20 at 6 24 21 pm

File: lib/ui/src/modules/ui/components/stories_panel/index.js [hunk 1]
```diff
@@ -84,6 +97,16 @@ StoriesPanel.propTypes = {
   openShortcutsHelp: PropTypes.func,
   name: PropTypes.string,
   url: PropTypes.string,
+  shortcutOptions: PropTypes.objectof(
```
Comment (Hypnosphi) [inline lib/ui/src/modules/ui/components/stories_panel/index.js]: typo: `objectOf`

But it's actually not needed here, it should be just `shape`

**Review by Hypnosphi — CHANGES_REQUESTED** [review]

**Review by Hypnosphi — APPROVED** [review]

**Pull request merged**